corehq/apps/app_manager/templates/app_manager/partials/modules/case_list_module_overwrite.html

```diff
@@ -4,44 +4,44 @@
 <form class="form-inline" method='POST' action='{% url "overwrite_module_case_list" domain app.id module.unique_id %}'>
   {% csrf_token %}
   <fieldset>
-    <label for="other_module_id">{% trans "Module" %}: </label>
-    <select name='dest_module_unique_ids' class='form-control multiselect-caselist' multiple="multiple">{% for other_mod in available_modules %}
-      <option value={{ other_mod.unique_id }}>{{ other_mod.name|html_trans:langs }}</option>
-    {% endfor %}</select>
-    <input name="detail_type" type="hidden" value="{{ detail_type }}" />
-    <br>
-    <div class="checkbox">
-      <label>
-        <input type="checkbox" name="display_properties"/>
-        Display Properties
-      </label><br>
-      <label>
-        <input type="checkbox" name="case_list_filter"/>
-        Case List Filter
-      </label><br>
-      <label>
-        <input type="checkbox" name="other_configuration"/>
-        Other Configuration
-      </label><br>
+    <label for="other_module_id">{% trans "Module(s)" %}: </label>
+    <div style="width:25%; display: inline-block">
+      <select name='dest_module_unique_ids' class='form-control multiselect-caselist' multiple="multiple" >{% for other_mod in available_modules %}
```

Comment:
Minor: this would be easier to read if the `{% for %}` and then `{% endfor %}` were each on their own lines.